<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color:rgb(177,198,200);</string>
  </property>
  <widget class="QPushButton" name="findPatientIDButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(144, 191, 200); font-size:14pt;</string>
   </property>
   <property name="text">
    <string>Find Patient ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findMedicineButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(144, 191, 200); font-size:14pt;</string>
   </property>
   <property name="text">
    <string>Find Medicine</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
font: 75 20pt &quot;Microsoft JhengHei Light&quot;;</string>
   </property>
   <property name="text">
    <string>MAIN MENU</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
